MSProject/ViewControllers/LibraryTableView/WBLibraryTableViewController.swift

```diff
@@ -138,6 +141,16 @@ class WBLibraryTableViewController: UIViewController {
         refreshControl?.tintColor = .woloxBackgroundColor()
         _view.bookTable.refreshControl = refreshControl
     }
+    
+    // MARK: - Spotlight
+    override func restoreUserActivityState(_ activity: NSUserActivity) {
+        if activity.activityType == CSSearchableItemActionType, let info = activity.userInfo, let selectedIdentifier = info[CSSearchableItemActivityIdentifier] as? String {
+            if let book = viewModel.getBookById(id: selectedIdentifier) {
+                let detailBookViewController = WBDetailBookViewController(with: book)
+                navigationController?.pushViewController(detailBookViewController, animated: true)
+            }
+        }
+    }
```

Comment:
you need to have the app open on background for this to work right? just curious

Reply:
yes, cause we have the login as first view, it will need a change on the appDelegate to avoid the login and push to LibraryTableViewController to go to detail book selected,  we can implement an 'unlogged' state in order to see the list but no able to wish, rent, comment or suggest book (?